src/pages/docs/approvals/jira-service-management/index.md

```diff
@@ -169,7 +169,7 @@ closed the above process will be followed again.
 Once an Issue has been found, the deployment will only proceed if the **State** of the CR is 
 `Implementing`. If the **State** is either `Preview`, `Planning`, `Authorize`, or `Awaiting 
 Implementation` the 
-deployment will wait. Any other **State** will cause the deployment task to fail.
+deployment will wait. Any other **State** will cause the deployment task to fail, with logs written to ServerLogs instead of Tasklogs.  
```

Comment:
Not sure the exact answer to this as I couldn't find an answer online... "ServerLogs" and "Tasklogs" - are these product-specific names or can we just use "server logs" and "task logs"?

If they are product-specific naming conventions... can we double check the consistency issue (i.e. I'd expect "ServerLogs" and "TaskLogs" or something like that.

```suggestion
deployment will wait. Any other **State** will cause the deployment task to fail, with logs written to server logs instead of task logs.
```